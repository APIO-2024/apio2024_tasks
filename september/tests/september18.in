txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 5 7 6 3 1 2 4
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 3 4 5 1 2
10 1
0 1 2 3 4 5 6 7 8
9 7 8 3 6 5 2 4 1
10 1
0 1 2 3 4 5 6 7 8
7 8 9 4 3 2 5 6 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 4 5 6 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 3 4 5 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 4 2 1 3
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 5 6 4 1 3 2
10 1
0 1 2 3 4 5 6 7 8
6 7 9 8 5 1 2 4 3
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
3 9 6 8 7 5 4 2 1
10 1
0 1 2 3 4 5 6 7 8
2 9 8 3 4 5 6 1 7
10 1
0 1 2 3 4 5 6 7 8
8 9 7 5 6 3 4 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 4 7 3 5 1 2
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 36 35 34 33 32 31 30 29 28 27 26 25 24 23 22 21 20 18 19 17 16 14 15 13 12 11 9 10 8 7 6 5 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 36 35 34 33 32 31 30 29 28 27 26 25 24 22 23 21 20 19 18 17 16 15 13 14 12 11 9 10 7 8 6 5 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 35 36 37 33 34 32 30 31 28 29 27 26 24 25 23 20 21 22 18 19 17 15 16 14 13 11 12 10 8 9 6 5 7 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 36 35 37 38 31 34 32 33 25 26 27 29 28 30 20 22 10 19 17 14 8 23 21 16 13 24 9 12 11 15 18 7 6 5 1 2 3 4
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 36 37 34 35 33 31 32 30 29 28 27 24 26 25 22 23 20 21 19 18 17 16 15 14 13 12 11 10 9 7 8 6 5 4 3 2 1
100 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98
85 64 81 90 93 78 97 67 89 84 65 99 94 86 79 77 69 62 71 75 95 96 63 87 83 91 82 88 66 74 80 98 72 70 73 68 92 76 27 60 14 2 61 43 39 20 35 46 30 23 59 53 57 9 32 49 19 34 21 10 38 44 7 50 45 29 8 1 28 13 3 40 52 6 36 37 11 41 18 33 42 51 56 31 54 25 4 17 15 12 55 5 26 22 24 16 58 48 47
1000 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255 256 257 258 259 260 261 262 263 264 265 266 267 268 269 270 271 272 273 274 275 276 277 278 279 280 281 282 283 284 285 286 287 288 289 290 291 292 293 294 295 296 297 298 299 300 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390 391 392 393 394 395 396 397 398 399 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 417 418 419 420 421 422 423 424 425 426 427 428 429 430 431 432 433 434 435 436 437 438 439 440 441 442 443 444 445 446 447 448 449 450 451 452 453 454 455 456 457 458 459 460 461 462 463 464 465 466 467 468 469 470 471 472 473 474 475 476 477 478 479 480 481 482 483 484 485 486 487 488 489 490 491 492 493 494 495 496 497 498 499 500 501 502 503 504 505 506 507 508 509 510 511 512 513 514 515 516 517 518 519 520 521 522 523 524 525 526 527 528 529 530 531 532 533 534 535 536 537 538 539 540 541 542 543 544 545 546 547 548 549 550 551 552 553 554 555 556 557 558 559 560 561 562 563 564 565 566 567 568 569 570 571 572 573 574 575 576 577 578 579 580 581 582 583 584 585 586 587 588 589 590 591 592 593 594 595 596 597 598 599 600 601 602 603 604 605 606 607 608 609 610 611 612 613 614 615 616 617 618 619 620 621 622 623 624 625 626 627 628 629 630 631 632 633 634 635 636 637 638 639 640 641 642 643 644 645 646 647 648 649 650 651 652 653 654 655 656 657 658 659 660 661 662 663 664 665 666 667 668 669 670 671 672 673 674 675 676 677 678 679 680 681 682 683 684 685 686 687 688 689 690 691 692 693 694 695 696 697 698 699 700 701 702 703 704 705 706 707 708 709 710 711 712 713 714 715 716 717 718 719 720 721 722 723 724 725 726 727 728 729 730 731 732 733 734 735 736 737 738 739 740 741 742 743 744 745 746 747 748 749 750 751 752 753 754 755 756 757 758 759 760 761 762 763 764 765 766 767 768 769 770 771 772 773 774 775 776 777 778 779 780 781 782 783 784 785 786 787 788 789 790 791 792 793 794 795 796 797 798 799 800 801 802 803 804 805 806 807 808 809 810 811 812 813 814 815 816 817 818 819 820 821 822 823 824 825 826 827 828 829 830 831 832 833 834 835 836 837 838 839 840 841 842 843 844 845 846 847 848 849 850 851 852 853 854 855 856 857 858 859 860 861 862 863 864 865 866 867 868 869 870 871 872 873 874 875 876 877 878 879 880 881 882 883 884 885 886 887 888 889 890 891 892 893 894 895 896 897 898 899 900 901 902 903 904 905 906 907 908 909 910 911 912 913 914 915 916 917 918 919 920 921 922 923 924 925 926 927 928 929 930 931 932 933 934 935 936 937 938 939 940 941 942 943 944 945 946 947 948 949 950 951 952 953 954 955 956 957 958 959 960 961 962 963 964 965 966 967 968 969 970 971 972 973 974 975 976 977 978 979 980 981 982 983 984 985 986 987 988 989 990 991 992 993 994 995 996 997 998
997 996 999 998 995 990 992 991 993 994 986 985 989 984 987 988 980 981 982 983 972 976 974 978 973 979 975 977 971 969 970 968 967 966 964 962 965 963 958 959 961 960 955 952 956 953 954 951 957 950 947 949 948 944 943 946 942 945 938 936 941 939 940 937 934 935 932 933 931 928 926 929 930 927 919 920 923 921 922 924 925 915 917 914 916 918 907 909 911 913 908 912 910 903 902 906 905 904 897 898 896 899 900 901 889 894 893 895 891 888 890 892 886 887 884 885 883 880 881 879 878 882 874 872 875 873 876 877 869 866 867 868 871 870 860 863 862 864 861 865 857 859 858 855 856 853 851 852 850 854 843 842 846 845 844 848 849 847 840 838 836 841 835 837 839 834 833 832 831 829 827 828 830 824 825 826 823 818 819 821 822 820 817 815 814 816 812 813 808 811 807 810 809 802 806 803 805 800 804 801 796 795 798 799 797 790 792 791 793 794 786 784 789 788 783 787 782 785 781 779 778 780 777 775 772 776 774 773 768 769 771 770 764 765 763 767 766 760 762 759 761 758 756 757 755 749 754 752 750 751 753 748 747 745 746 742 744 743 741 740 734 736 739 738 737 735 733 732 731 730 728 727 729 726 721 722 723 725 724 720 718 719 714 715 717 716 712 709 713 710 711 702 705 706 707 701 704 703 708 700 699 696 697 698 695 693 694 692 691 686 688 687 689 684 685 690 681 683 677 679 678 680 682 673 676 675 672 674 666 669 668 670 671 667 661 662 665 664 663 658 659 660 657 654 656 653 655 647 648 651 652 650 646 649 644 641 645 643 642 637 639 636 638 640 635 631 633 634 632 630 627 626 629 628 625 624 623 622 621 618 620 619 617 616 613 614 615 611 607 608 612 609 610 606 600 602 599 604 601 605 603 596 592 598 593 595 594 597 591 586 587 590 588 589 579 582 583 584 585 581 580 578 576 575 577 573 571 574 572 570 567 568 569 564 563 565 566 561 562 560 559 558 556 557 553 550 555 554 552 549 551 543 548 544 545 546 547 540 537 536 539 542 538 541 532 534 533 531 530 535 529 528 527 526 524 525 523 522 521 520 519 514 515 518 513 517 516 512 510 509 511 503 507 502 505 506 504 508 497 495 499 501 496 498 500 489 492 493 490 491 494 488 487 484 483 482 485 486 481 478 477 480 479 474 472 473 475 476 471 470 466 468 467 469 465 460 462 463 461 464 456 457 459 458 451 454 452 455 453 449 450 443 445 446 444 442 447 448 440 439 441 435 437 434 436 432 433 438 431 426 430 429 428 427 425 423 424 421 420 422 413 414 418 415 417 419 416 412 411 408 407 410 409 404 406 405 399 401 402 400 403 397 398 394 393 396 392 395 389 391 390 388 387 386 382 380 381 378 385 384 383 379 375 376 372 374 377 371 373 367 369 364 365 368 366 370 363 362 361 356 359 358 357 360 351 349 348 354 352 350 355 353 346 345 344 341 340 347 342 343 336 337 339 335 338 332 329 328 330 333 331 334 326 327 323 325 324 322 320 319 317 318 321 315 316 314 313 310 309 312 311 308 306 307 302 305 301 304 303 295 296 297 300 299 294 298 292 293 290 288 291 289 287 283 284 285 286 280 278 282 281 279 276 277 274 275 273 271 272 269 270 267 268 265 259 261 263 266 260 264 262 258 257 252 254 253 255 250 249 251 256 248 247 245 246 238 242 237 240 244 243 239 241 233 235 236 234 232 231 230 229 227 228 223 224 226 222 221 220 225 219 218 217 216 214 215 213 212 209 208 207 206 210 211 205 203 204 196 202 197 201 200 199 198 190 193 192 194 191 195 189 187 188 186 185 181 178 180 179 183 184 182 176 177 175 174 171 173 172 168 170 167 166 169 161 162 163 159 165 164 160 158 157 154 156 155 153 148 152 151 147 146 149 150 142 144 143 145 140 139 141 135 136 137 138 133 134 128 129 126 130 132 127 131 125 119 117 120 118 123 121 122 124 116 115 113 114 111 109 112 110 104 103 102 106 107 108 101 105 100 99 98 93 94 92 95 97 96 87 88 90 89 91 85 86 79 82 81 83 84 80 72 77 78 76 73 74 75 66 65 71 69 70 68 67 60 63 64 62 59 61 58 56 57 51 52 54 53 48 55 50 49 47 46 45 44 42 37 36 40 38 41 39 43 35 32 29 33 30 34 31 28 24 23 27 26 25 21 22 20 19 18 16 14 17 15 12 13 11 5 6 9 4 10 7 3 8 1 2
100 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98
98 99 97 96 93 95 94 92 91 89 90 88 87 86 83 85 84 82 80 79 81 78 76 74 77 75 72 71 70 73 68 69 67 66 63 64 65 62 61 60 58 59 57 56 55 53 54 52 51 49 50 47 48 46 42 45 44 43 38 39 41 40 37 36 35 34 32 33 31 30 28 27 29 25 26 24 23 21 22 20 18 19 17 16 15 14 12 13 11 9 10 8 6 5 7 4 2 3 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
36 35 29 39 33 37 30 34 32 38 31 28 26 27 23 21 25 22 24 12 19 11 18 8 17 20 15 9 14 16 13 10 7 6 4 5 1 2 3
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 37 33 36 35 34 32 26 27 31 30 28 29 21 22 25 23 24 20 19 18 16 17 15 14 10 11 13 8 12 9 7 6 5 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 35 36 33 34 31 32 30 29 27 28 26 25 24 23 21 22 19 20 18 17 16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 36 38 37 34 35 25 30 26 31 28 29 33 27 32 22 21 23 19 20 24 17 16 15 18 9 13 8 14 12 10 11 4 5 3 7 1 2 6
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 37 39 36 34 35 32 33 31 30 29 28 26 27 24 25 22 23 21 20 19 18 16 17 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
7 8 9 6 4 5 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 8 5 7 6 3 4 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
7 8 9 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
6 9 5 7 8 1 2 4 3
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 4 3 5 2 1
10 1
0 1 2 3 4 5 6 7 8
7 8 9 6 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
7 8 9 3 2 6 5 1 4
10 1
0 1 2 3 4 5 6 7 8
8 9 7 5 6 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 5 6 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
4 6 8 5 2 9 7 3 1
10 1
0 1 2 3 4 5 6 7 8
9 3 8 6 7 5 4 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 4 6 5 7 3 1 2
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 4 3 2 1
