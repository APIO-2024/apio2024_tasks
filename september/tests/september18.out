7ckgnn4wyi495puj3ibqf81dqvapyv6b
8
8
3
7
8
5
4
3
7
7
8
6
9
5
3
8
3
1
6
4
35
34
25
9
30
2
309
61
9
21
32
8
31
5
6
8
8
8
7
7
8
8
2
7
7
6
2
7
7
2
3
5
8
